package.json

```diff
@@ -1,15 +1,17 @@
 {
   "name": "@tbd54566975/dwn-sdk-js",
-  "version": "0.0.17",
+  "version": "0.0.18",
   "description": "A reference implementation of https://identity.foundation/decentralized-web-node/spec/",
   "type": "module",
   "types": "./dist/esm/src/index.d.ts",
   "engines": {
     "node": ">= 16"
   },
+  "main": "./dist/cjs/src/index.js",
```

Comment:
For my learning: what's the reason for bringing back building cjs? Is this something required by react native?

Reply:
@thehenrytsai related:

https://github.com/facebook/metro/issues/670

Reply:
RN **does** use ESM but for whatever reason just doesn't support the `exports` field yet

Reply:
@thehenrytsai i felt bad for including a `main` field that wasn't `cjs` because that is a widely accepted convention.

Reply:
K, based on the parallel conversations, I take it that `cjs` is now needed not because of metro/react, but because of electron. Got it.